types/better-scroll/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+      "module": "commonjs",
+      "lib": [
+          "es6",
+          "dom"
+      ],
+      "noImplicitAny": true,
+      "noImplicitThis": true,
+      "strictNullChecks": false,
```

Comment:
All new declarations need to have `strictNullChecks` set to true.

Reply:
@DanielRosenwasser ok.

Reply:
@DanielRosenwasser had changed as your comments